src/ccpa/sourcepoint.test.js

```diff
@@ -1,15 +1,51 @@
+/* eslint-disable no-underscore-dangle */
 import http from 'http';
 import url from 'url';
 import { init } from './sourcepoint';
+import { ACCOUNT_ID } from '../lib/accountId';
+
+jest.mock('../onConsentChange', () => ({
+	invokeCallbacks: jest.fn(),
+}));
 
 describe('Sourcepoint', () => {
-	init();
+	afterEach(() => {
+		window._sp_ccpa = undefined;
+	});
+
+	it('should throw error if window scp exists', () => {
+		// eslint-disable-next-line no-underscore-dangle
```

Comment:
```suggestion
```

Reply:
oh wow vscode  auto save linter puts back automatically when i remove it 😳